dynawo/sources/Models/Modelica/Dynawo/Electrical/Loads/LoadAlphaBeta.extvar

```diff
@@ -16,5 +16,9 @@
   <variable type="continuous" id="terminal.V.im"/>
   <variable type="discrete" id="PRefPu.value"/>
   <variable type="discrete" id="QRefPu.value"/>
+<!--
   <xi:include href="./SwitchOffLoad.extvar" xpointer="xmlns(dyn=http://www.rte-france.com/dynawo)xpointer(//dyn:external_variables/*)"/>
```

Comment:
xpointer not working on windows ?

Reply:
Frederic had  issues until he found out that xmlint is the tool taking care of xpointers. As we already talked about removing those cross references to solve the random compilation crash we decided to do it at this time.

Reply:
Ok so we have to remove it entirely